download-maven-plugin/src/main/java/com/googlecode/WGet.java

```diff
@@ -108,6 +110,20 @@
   private boolean unpack;
 
   /**
+   * Custom username for the download
+   *
+   * @parameter default-value=""
```

Comment:
What about removing the "default-value" so the default value is null instead of blank?